storage/pebble/operation/events_test.go

```diff
@@ -16,7 +15,7 @@ import (
 // TestRetrieveEventByBlockIDTxID tests event insertion, event retrieval by block id, block id and transaction id,
 // and block id and event type
 func TestRetrieveEventByBlockIDTxID(t *testing.T) {
```

Comment:
## ⚠️  follow-up to [my comment above](https://github.com/onflow/flow-go/pull/6197#issuecomment-2278752293):

This test case can be modified here to show that the [`traverse` function](https://github.com/onflow/flow-go/blob/51969ecf92ea9775622890d6be9be223f6f7a593/storage/pebble/operation/common.go#L271) is indeed broken. At the moment, the test simply does not cover the boundary condition of a transactionID being `FF...F`: https://github.com/onflow/flow-go/blob/51969ecf92ea9775622890d6be9be223f6f7a593/storage/pebble/operation/events_test.go#L22

If you replace [line 22](https://github.com/onflow/flow-go/blob/51969ecf92ea9775622890d6be9be223f6f7a593/storage/pebble/operation/events_test.go#L22) with the following code, the test fails because only one of the two stored events are retrieved:

```suggestion
txIDs := []flow.Identifier{
  flow.HashToID([]byte{0x11}),
  [32]byte{0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF},
}
```

Reply:
Fixed by https://github.com/onflow/flow-go/pull/6316. 

Your test case are added and passed.